deploy/charts/harvester-crd/templates/harvesterhci.io_schedulevmbackups.yaml

```
@@ -0,0 +1,196 @@
+---
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    {}
+  name: schedulevmbackups.harvesterhci.io
+spec:
+  group: harvesterhci.io
+  names:
+    kind: ScheduleVMBackup
+    listKind: ScheduleVMBackupList
+    plural: schedulevmbackups
+    shortNames:
+    - svmbackup
+    - svmbackup
```

I suspect one of the above shortNames is intended to be plural:

```suggestion
    - svmbackups
```